jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1098,6 +1098,30 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _sort(*operand: tf.Tensor, dimension: int = -1,
+          is_stable: bool = True) -> Tuple[TfVal, ...]:
```

Comment:
We prefer to not use default values, we want errors if the argument is missing. I think that the `sort_p` primitive always has the `dimension` and `is_stable`. I propose
```
def _sort(*operand: TfVal, *, dimension: int, is_stable: bool)
```